### DISCUSSION TITLE
Supabase oAuth flutter

### DISCUSSION THREAD

**Discussion by andim27** [supabase/supabase#14764] [Feature Requests]
![records_facebook](https://github.com/supabase/supabase/assets/4599713/65c828c2-0e8a-4a6a-ba2a-5778b8d7bd8c)
![facebook_redirect](https://github.com/supabase/supabase/assets/4599713/c7fe9886-bc1b-4c44-b380-89f1844271dc)
I use signInWithOAuth(Provider.facebook) according to documentation.Record created - all is ok but redirect screen does not close after registration on android.

**Comment by dshukertjr** [comment]
Hi there, I can see that you have opened an issue with the same content, so I'm going to close this discussion in favor of the issue. 
https://github.com/supabase/supabase-flutter/issues/503
